pandas/core/groupby/groupby.py

```diff
@@ -3272,23 +3275,38 @@ def first(
         ...     )
         ... )
         >>> df["D"] = pd.to_datetime(df["D"])
+
         >>> df.groupby("A").first()
-             B  C          D
+            B  C          D
         A
         1  5.0  1 2000-03-11
         3  6.0  3 2000-03-13
+
+        >>> df.groupby("A").nth(0)
+            B  C          D
+        A
+        1  NaN  1 2000-03-11
+        3  6.0  3 2000-03-13
```

Comment:
I think this page should only include documentation on `first`. Can you remove the use of other methods.